rusk-wallet/src/main.rs

```diff
@@ -211,16 +205,17 @@ struct Rusk {
 }
 
 /// Connect to rusk services via TCP
-async fn rusk_tcp(rusk_addr: String, prov_addr: String) -> Result<Rusk, Error> {
+async fn rusk_tcp(rusk_addr: &str, prov_addr: &str) -> Result<Rusk, Error> {
```

Comment:
Looks a bit off that we take `&str` as argument to `.to_string` later on. However, it is more appropriate to receive `&str` as arguments in this type of configuration function.

Looks like the flaw is in `NetworkClient::connect` that expects a `String`, and not a `&str`

Reply:
I agree, that's why I changed the fn header